src/runtime/tests/manifest-test.ts

```diff
@@ -606,6 +606,16 @@ ${particleStr1}
     verify(manifest);
     verify(await parseManifest(manifest.toString()));
   });
+  it('parses meta namespace section', async () => {
+    const manifest = await parseManifest(`
+      meta
+        name: 'Awesome Arc'
```

Comment:
We should consider passing the manifest name as metadata to the plan classes for debugging purposes (later).